pkg/analysis_server/lib/src/services/correction/dart/add_missing_switch_cases.dart

```diff
@@ -70,11 +70,8 @@ class AddMissingSwitchCases extends ResolvedCorrectionProducer {
         builder.write(location.prefix);
         builder.write(lineIndent);
         builder.write(singleIndent);
-        builder.writeln('// TODO: Handle this case.');
```

Comment:
Using `UnimplementedError` is seems a good idea to me.
But I will also keep the `// TODO ...` generating code.